### DISCUSSION TITLE
How i combine Latex with Quarto?

### DISCUSSION THREAD

**Discussion by jvictorferreira3301** [quarto-dev/quarto-cli#8149] [Q&A] [answered]
### Description

I created a complete book in LaTeX and planned to publish it on the internet. Initially, my idea was to convert the LaTeX file to HTML using make4ht/text4ht and host it on GitHub Pages. However, during my research, I came across Quarto. How could I take my latex production and use the Quarto to make a website for the book? What I did was take the output of the tex --> html conversion and change the html extension to qmd and use it as input in the Quarto, but this leads to some incompatibilities, is there an easier way for me to do this?

**Comment by mcanouil** [comment, marked as answer]
Quarto is based on markdown.
See the guides and start with that:
- https://quarto.org/docs/get-started/
- https://quarto.org/docs/books/

The best way would be to convert to markdown then modify to get Quarto features.

```bash
quarto pandoc your-book.latex --from latex --to markdown -o your-book.qmd
```

**Reply by danmackinlay** [reply]
If you are like me you have made heavy use of macros in your LaTeX and you might find the resulting HTML does not render very well. Some colleagues of mine recommend [plastex](http://plastex.github.io/plastex/) for this eventuality. I have not yet tried it myself.
